lib/vanagon/platform/windows.rb

```diff
@@ -55,6 +56,7 @@ def generate_packaging_artifacts(workdir, name, binding)
       def generate_msi_packaging_artifacts(workdir, name, binding)
         FileUtils.mkdir_p(File.join(workdir, "wix"))
         erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/service.component.wxs.erb"), File.join(workdir, "wix", "service.#{name}.wxs"), false, { :binding => binding })
+        erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/service-filter.xslt.erb"), File.join(workdir, "wix", "#{name}.service-filter.xslt"), false, { :binding => binding })
```

Comment:
If we're renaming this file, we should change it from `project.service-filter.xslt.erb` to `#{name}.service-filter.xslt`. Rather than adding a new value to the name, we should be replacing a value.